<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="push10button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push50button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push100button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push500button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Buy_a_Coffee">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Buy_a_Tea">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Buy_a_Milk">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Milk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RESET">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>91</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
